Windows/csharp_samples/csharp_samples.csproj

```diff
@@ -78,11 +79,11 @@
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="packages\Microsoft.CognitiveServices.Speech.0.2.12733\build\net45\Microsoft.CognitiveServices.Speech.targets" Condition="Exists('packages\Microsoft.CognitiveServices.Speech.0.2.12733\build\net45\Microsoft.CognitiveServices.Speech.targets')" />
+  <Import Project="packages\Microsoft.CognitiveServices.Speech.0.4.0\build\net45\Microsoft.CognitiveServices.Speech.targets" Condition="Exists('packages\Microsoft.CognitiveServices.Speech.0.4.0\build\net45\Microsoft.CognitiveServices.Speech.targets')" />
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('packages\Microsoft.CognitiveServices.Speech.0.2.12733\build\net45\Microsoft.CognitiveServices.Speech.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Microsoft.CognitiveServices.Speech.0.2.12733\build\net45\Microsoft.CognitiveServices.Speech.targets'))" />
+    <Error Condition="!Exists('packages\Microsoft.CognitiveServices.Speech.0.4.0\build\net45\Microsoft.CognitiveServices.Speech.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Microsoft.CognitiveServices.Speech.0.4.0\build\net45\Microsoft.CognitiveServices.Speech.targets'))" />
```

Comment:
>.0.4.0 [](start = 74, length = 6)

is this actually what will be the release nuget??

Reply:
yes

Reply:
yes

---
In reply to: [193502396](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193502396) [](ancestors = 193502396)